### PR TITLE
Add Elasticsearch debugging guide #2830

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
Add Elasticsearch debugging guide #2830

### CODE REVIEW

File: source/developers/projects/search/index.rst [hunk 1]
```diff
@@ -13,6 +13,9 @@ Crafter Search
 
 Crafter Search is the search facade for Crafter CMS. Crafter Search proxies the real search engine deployed behind it.
 
+.. note::
+    Crafter Search is supported for backwards compatibility.  Crafter CMS's default search engine is Elasticsearch.
```
Comment (sumerjabri) [inline source/developers/projects/search/index.rst]: backward, not backwards

File: source/includes/how-search-works-solr.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ^^^^
 Flow
 ^^^^
-.. image:: /_static/images/search-mechanics.gif
```
Comment (sumerjabri) [inline source/includes/how-search-works-solr.rst]: Please delete the binary if no longer in use.

File: source/system-administrators/activities/debugging-search-elasticsearch.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  export ES_HOST=${ES_HOST:="localhost"}
+  export ES_PORT=${ES_PORT:="9201"}
+
+|
+
+----------------------------------------------
+Configure Crafter Studio: Crafter Deployer URL
+----------------------------------------------
+
+``$CATALINA_HOME/shared/classes/crafter/studio/extension/studio-config-override.yaml``
```
Comment (avasquez614) [inline source/system-administrators/activities/debugging-search-elasticsearch.rst]: Now the Deployer URL can be changed in the `crafter-setenv.sh` directly without touching the Studio YAML: https://github.com/craftercms/craftercms/blob/be1833214f07f40913f70c086c22412972f21a1b/resources/env/authoring/bin/crafter-setenv.sh#L49

File: source/system-administrators/activities/debugging-search-solr.rst [hunk 4]
```diff
@@ -97,28 +100,15 @@ Configure Crafter Studio: Crafter Deployer URL
...
-
-----------------------------------
-Configure Crafter Search: Solr URL
-----------------------------------
-
-``$CATALINA_HOME/shared/classes/crafter/search/extension/server-config.properties``
+  ############################################################
+  ##                    Preview Deployer                    ##
+  ############################################################
+  # Default preview deployer URL (can be overridden per site)
```
Comment (avasquez614) [inline source/system-administrators/activities/debugging-search-solr.rst]: Now the Deployer URL can be changed in the `crafter-setenv.sh` directly without touching the Studio YAML: https://github.com/craftercms/craftercms/blob/be1833214f07f40913f70c086c22412972f21a1b/resources/env/authoring/bin/crafter-setenv.sh#L49

**Pull request merged**